<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <widget class="QWidget" name="device">
        <attribute name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/resources/icons/sys_icon_options.png</normaloff>:/icons/resources/icons/sys_icon_options.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Device</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QScrollArea" name="deviceScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>945</width>
              <height>418</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Sound</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="TouchSpinBox" name="soundWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Brightness</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="TouchSpinBox" name="brightnessWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Language</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="TouchSpinBoxString" name="languageWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Color scheme</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="TouchSpinBoxString" name="colorSchemeWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>48</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>164</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ascan">
        <attribute name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/resources/icons/sys_icon_zoom_in.png</normaloff>:/icons/resources/icons/sys_icon_zoom_in.png</iconset>
        </attribute>
        <attribute name="title">
         <string>A-Scan</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QScrollArea" name="ascanScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>945</width>
              <height>418</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bscan">
        <attribute name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/resources/icons/sys_icon_zoom.png</normaloff>:/icons/resources/icons/sys_icon_zoom.png</iconset>
        </attribute>
        <attribute name="title">
         <string>B-Scan</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QScrollArea" name="bscanScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>945</width>
              <height>418</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/resources/icons/sys_icon_command_line.png</normaloff>:/icons/resources/icons/sys_icon_command_line.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Debug</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QScrollArea" name="debugScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>945</width>
              <height>418</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="showFps">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QCheckBox::indicator {
     width: 20px;
     height: 20px;
}</string>
                 </property>
                 <property name="text">
                  <string>Show A-Scan FPS</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchSpinBox</class>
   <extends>QWidget</extends>
   <header>ui/controls/touchspinbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TouchSpinBoxString</class>
   <extends>QWidget</extends>
   <header>ui/controls/touchspinboxstring.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
